tests/test_memory.py

```diff
@@ -126,3 +126,7 @@ def test_mstore_mload_reflexivity():
 
     ret = run(code)
     assert int.from_bytes(ret, 'big') == 0xff112233445566778899aabbccddeeff
+
+
+def test_failing()
+    assert True
```

Comment:
what's that for?

Reply:
i was testing it in my ci just to make sure it reported correctly!